### PR TITLE
feat(cleanup): add cleanup script

### PR DESCRIPTION
**Pull request by WikiRik**
I just copied this together, not sure if it actually works though
@serge1peshcoff can you test this out some time?

### CODE REVIEW

File: scripts/myaegeecleanup.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            });
+        } catch (err) {
+            robot.logger.error(err);
+            return msg.reply(`Could not establish SSH connection: ${err}`);
+        }
+
+        msg.reply('The clean up has started.');
+
+        try {
+            const result = await ssh.execCommand('docker system prune');
```
Comment (serge1peshcoff) [inline scripts/myaegeecleanup.js:59]: this won't work I think without `--yes`

Comment (WikiRik) [reply scripts/myaegeecleanup.js]: -f indeed needed to be added, as per https://docs.docker.com/config/pruning/

**Review by serge1peshcoff — APPROVED** [review]
should work I think, cannot test it right now unfortunately

**Comment by WikiRik** [comment]
No worries, just test it out when you can

**Comment by WikiRik** [comment]
build won't run now because of issues with Circle CI. But I think it's good if we merge this soon since we have run out of disk space twice already this week

**Review by serge1peshcoff — APPROVED** [review]
should work in theory, let's merge and fix if something's wrong

**Pull request merged**